### ISSUE TITLE
🐛 [Bug Report] (PVP)- taking dots/faeri fire doees not break stealth.

### ISSUE THREAD

**Issue by oldspices** [Yatzii93/TBC5MAN-Bug-Tracker#83]
# Decription: #
taking dots/faeri fire doees not break stealth.
# How it works: #

# How it should work: #
casting dots on target or feri fire which makes stealth not possible- should break stealth. 
# Source - You need to provide a source as evidence: #
3:55 https://www.youtube.com/watch?v=2CMLa2enhG4 you can see him dotting  breaks the two out of stealth 
# Additional Information: #

**Comment by Yatzii93** [comment]
Taking dot damage 100% is breaking stealth.

**Comment by oldspices** [comment]
yes, the application should removee stealth as shown in that video- as well as, applying faeri fire, which in its tooltip says it prevents stealth- which is a double whammy. Currently if a rogue is near you in stealth and you Faeri fire him, he remains in steatlh. (as well as fear/blind/etc)
